metricflow/plan_conversion/dataflow_to_sql.py

```diff
@@ -615,6 +628,43 @@ def visit_compute_metrics_node(self, node: ComputeMetricsNode) -> SqlDataSet:
             elif metric.type is MetricType.DERIVED:
                 assert metric.type_params.expr
                 metric_expr = SqlStringExpression(sql_expr=metric.type_params.expr)
+            elif metric.type == MetricType.CONVERSION:
+                conversion_type_params = metric.type_params.conversion_type_params
+                assert (
+                    conversion_type_params
+                ), "A conversion metric should have type_params.conversion_type_params defined."
+                base_measure = conversion_type_params.base_measure
+                conversion_measure = conversion_type_params.conversion_measure
+                base_measure_column = self._column_association_resolver.resolve_spec(
+                    MeasureSpec(element_name=base_measure.post_aggregation_measure_reference.element_name)
+                ).column_name
+                conversion_measure_column = self._column_association_resolver.resolve_spec(
+                    MeasureSpec(element_name=conversion_measure.post_aggregation_measure_reference.element_name)
+                ).column_name
+
+                calculation_type = conversion_type_params.calculation
+                if calculation_type == ConversionCalculationType.CONVERSION_RATE:
+                    metric_expr = SqlRatioComputationExpression(
+                        numerator=SqlColumnReferenceExpression(
+                            SqlColumnReference(
+                                table_alias=from_data_set_alias,
+                                column_name=conversion_measure_column,
+                            )
+                        ),
+                        denominator=SqlColumnReferenceExpression(
+                            SqlColumnReference(
+                                table_alias=from_data_set_alias,
+                                column_name=base_measure_column,
+                            )
+                        ),
+                    )
+                elif calculation_type == ConversionCalculationType.CONVERSIONS:
+                    metric_expr = SqlColumnReferenceExpression(
```

Comment:
Can we separate the logic for this column into a variable to reuse here and above? Unless you have an opinion about it being more readable this way - I generally prefer the dry-er code